apps/cli/src/sky/writeToFetchedMachineFile.ts

```diff
@@ -32,7 +32,7 @@ export const writeSkyConfig = async (opts: {
   }
   const prettierConfig = await prettier.resolveConfig(filePath);
   const pathToSave = getPathToSave(filePath);
-  const code = removeLeadingExport(skyConfig.configString);
+  const code = removeLeadingExport(skyConfig.prettyConfigString);
```

Comment:
@mellson check if this is correct?

Reply:
It is not, it should remain `configString`

Reply:
Thy property doesn't exit on the sky config type.

Reply:
Sounds like you might be on an older version of the `@statelyai/sky` package. Try running `yarn i` at the root of the repo, this should install a version where `configString` is there.